src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx

```diff
@@ -0,0 +1,189 @@
+</figcaption>
+
+<Steps>
+<Step>
+### Install the infrastructure agent [#infra-install]
+
+You can install the infrastructure agent two different ways: 
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview). 
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/). 
+</Step>
+<Step>
+###  Install the PostgreSQL quickstart
+
+1. Check out our PostgreSQL agent requirements in our [documentation](/docs/infrastructure/host-integrations/host-integrations-list/postgresql/postgresql-integration/)  before installing the agent.
+2. Open the [PostgreSQL quickstart installation](https://newrelic.com/instant-observability/postresql).
+3. Click  **Install now** to start the Postgre quickstart installation.
+4. Or you could execute the following command:
+
+```bash
+curl -Ls https://download.newrelic.com/install/newrelic-cli/scripts/install.sh | bash && sudo  NEW_RELIC_API_KEY=YOUR_KEY NEW_RELIC_ACCOUNT_ID=ACCOUNT_ID /usr/local/bin/newrelic install -n postgres-open-source-integration
+```
+</Step>
+
+<Step>
+### Integrate SonarQube with New Relic [#integrate]
+
+<CollapserGroup>
+<Collapser
+    id="step1"
+    title="Create a user token"
+    >
+
+1. Login into **SonarQube**, select **My Account**, then click **Security**.
+2. Name your token. Make sure that you've selected the type as `user token`.
+3. Copy the created token and save it in a file of your desired path.
+</Collapser>
+<Collapser
+    id="step2"
+    title="Configure NRI-Prometheus"
+    >
+1. Create a file named `nri-prometheus-config.yml` using this path: `/etc/newrelic-infra/integrations.d/nri-prometheus-config.yml`
+2. Use our [configuration template](https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample) to update the created file named `nri-prometheus-config.yml`.
+3. Update these fields:
+
+    * cluster_name: `YOUR_DESIRED_CLUSTER_NAME`
+    * bearer_token_file: `USER_TOKEN_FILE_PATH`
+    * urls: `[“https://YOUR_HOST_IP:9000/api/monitoring/metrics”]`
+    * Insecure_skip_verify: `true`
+
+Your `nri-prometheus-config.yml` file should look like this:
+
+```yml
```

Comment:
please use the below reference link to cross check the indentation of this yml.
https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample
Thank you 👍